charts/incubator/llama-gpt/values.yaml

```diff
@@ -0,0 +1,79 @@
+image:
+  repository: tccr.io/truecharts/llama-gpt-ui
+  pullPolicy: IfNotPresent
+  tag: v1.0.1@sha256:3e66190591adaf5bd91a0707c385bf0c26978c2627691659269335a105c93e7a
+apiImage:
+  repository: tccr.io/truecharts/llama-gpt-api
+  pullPolicy: IfNotPresent
+  tag: v1.0.1@sha256:39147d1e7829feb5c82e7654812ed629ee6b7a3422608597f64b232656fa8d83
+
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+    capabilities:
+      add:
+        - IPC_LOCK
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 3000
+        port: 3000
+  api:
+    enabled: true
+    type: ClusterIP
+    targetSelector: api
+    ports:
+      api:
+        enabled: true
+        targetSelector: api
+        port: 8000
```

Comment:
You probably need targetPort here...

Reply:
i thought setting port here does the same for targetting port?

Reply:
Yes it does. But changing the `port` will also change the `targetPort`
and in this case the `targetPort` will point to nowhere.
Its only effective when there is a variable to link it with.

While in this case is a clusterIP service and no "user defined"
Helm users can still expose it or do whatever!